dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -152,6 +169,24 @@ def self.regenerate_pdfs(scripts)
       end
     end
 
+    def self.add_metadata_to_ai_s3(file_path, metadata)
+      full_metadata = {
+        metadataAttributes: metadata
+      }
+      File.write(file_path, full_metadata.to_json)
+      flat_filename = File.basename(file_path)
+      data = File.read(file_path)
```

Comment:
This way should still work even if the file write to the local filesystem fails for some reason. Do we even need to write the file to the local filesystem in the first place?

```suggestion
      data = full_metadata.to_json
```

Reply:
Ah, good point!  I will streamline that.